modal_proto/api.proto

```diff
@@ -772,6 +772,15 @@ message ContainerFileFlushRequest {
   string file_descriptor = 1;
 }
 
+message ContainerFileLsRequest {
+  string path = 1;
+}
+
+message ContainerFileMkdirRequest {
+  string path = 1;
+  bool parents = 2;
```

Comment:
maybe `make_parents` would be more obvious

Reply:
The naming mostly comes from here: https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir. 
Does that work? Don't feel too strongly about this, as it's a proto anyways

Reply:
I don't feel strongly either - this is fine; it's just internal terminology that isn't obvious to the reader IMO